modules/ob-about-build-strategies.adoc

```diff
@@ -8,19 +8,19 @@
 
 The `BuildStrategy` and `ClusterBuildStrategy` resources define a series of steps to assemble an application. You can use the `BuildStrategy` resources within a namespace and the `ClusterBuildStrategy` resources within a cluster.
 
-The specification of a `BuildStrategy` or `ClusterBuildStrategy` resource consists of a `steps` object. The following example shows the specification of the `buildpacks v3` cluster build strategy:
+The specification of a `BuildStrategy` or `ClusterBuildStrategy` resource consists of a `steps` object. The following example shows the specification of the `buildah` cluster build strategy:
 
 [source,yaml]
 ----
 apiVersion: shipwright.io/v1beta1
 kind: ClusterBuildStrategy
 metadata:
-  name: buildpacks-v3
+  name: buildah
 spec:
   steps:
     - name: build-and-push
       image: docker.io/paketobuildpacks/builder-full:latest
-      env: 
+      env:
         - name: CNB_PLATFORM_API
           value: $(params.platform-api-version)
```

Comment:
@apoorvajagtap, Need your input to update this example with buildah.

Reply:
@shipsing the complete yaml needs to be replaced with the following:
~~~
apiVersion: shipwright.io/v1beta1
kind: ClusterBuildStrategy
metadata:
  name: buildah
spec:
  steps:
    - name: build-and-push
      image: quay.io/containers/buildah:v1.31.0
      workingDir: $(params.shp-source-root)
      command:
        - /bin/bash
       # ...
# ...
~~~